claim/validations.py

```diff
@@ -54,8 +54,8 @@ def validate_claim(claim, check_max):
     if len(errors) == 0:
         errors += validate_family(claim, claim.insuree)
     if len(errors) == 0:
-        validate_claimitems(claim)
-        validate_claimservices(claim)
+        errors += validate_claimitems(claim)
```

Comment:
Why do you add validation of claims and items to the errors? If I understand correctly this rule engine is supposed to set rejection reason and status on claim (or particular items and services). Are those errors used later on? If I remember correctly mutations on returning errors are failing, which shouldn't be occur in case of rejected items and services (as rejection is part of the workflow, not actual error).

Reply:
I decided to append current errors with service and items errors because it looked like the code was made to be that way. For instance couple of lines below the code searches for errors that were caused by reaching the limits of services but it never worked because the errors were never returned. We can discus the correct behavior on slack.